packages/webdriver/src/utils.js

```diff
@@ -382,3 +396,35 @@ export function setupDirectConnect(params) {
         params.path = directConnectPath
     }
 }
+
+/**
+ * get human readable message from response error
+ * @param {Error} err response error
+ */
+export const getSessionError = (err) => {
+    if (!err.message) {
+        return 'See logs for more information.'
+    }
+
+    // browser driver / service is not started
+    if (err.code === 'ECONNREFUSED') {
+        return `Unable to connect to "${err.address}:${err.port}", make sure browser driver is running on that address.`
+    }
+
+    // wrong path: selenium-standalone
+    if (err.message.includes('Whoops! The URL specified routes to this help page.')) {
+        return 'Make sure there is no `path` in wdio.conf!'
+    }
+
+    // wrong path: chromedriver, geckodriver, etc
+    if (BROWSER_DRIVER_ERRORS.some(m => err.message.includes(m))) {
+        return "Make sure `path: '/'` exists in config!"
+    }
+
+    // edge driver on localhost
+    if (err.message.includes('Bad Request - Invalid Hostname') && err.message.includes('HTTP Error 400')) {
+        return 'Run browser driver on 127.0.0.1 instead of localhost, ex: --host=127.0.0.1'
```

Comment:
The WebDriver option I think you are referring here is `hostname`

Reply:
for edge it's --host
![image](https://user-images.githubusercontent.com/25589559/67667340-e0379000-f96d-11e9-8abe-adde070ad331.png)

Reply:
edge returns ugly html page if host is 127.0.0.1 instead of localhost by default